org-bullets.el

```diff
@@ -46,80 +50,123 @@
     ;; ► • ★ ▸
     )
   "This variable contains the list of bullets.
+
 It can contain any number of symbols, which will be repeated."
   :group 'org-bullets
   :type '(repeat (string :tag "Bullet character")))
 
 (defcustom org-bullets-face-name nil
-  "This variable allows the org-mode bullets face to be
- overridden. If set to a name of a face, that face will be
- used. Otherwise the face of the heading level will be used."
+  "Face to override `org-mode' bullets face.
+
+If set to a name of a face, that face will be used.  When nil, do
+not change the face used."
+  :group 'org-bullets
+  :type '(choice (const :tag "Off" nil)
```

Comment:
Is the added complexity of providing two different behaviors here worth it? Why would a user choose one option instead of the other?

Reply:
This is the same behaviour. Nothing changed except I specified a better customization type.  This allows auto-completion of faces.

Reply:
The fact that faces are symbols doesn't make the variable hold all type of symbols.  Nil is also a (constant) symbol.

In order to provide completion for both nil and symbol faces, I need to make a [composite type](https://www.gnu.org/software/emacs/manual/html_node/elisp/Composite-Types.html#Composite-Types).

However, this does not make the variable typed or restrict in any way setting the variable to any other types.  `:type` keywords in `defcustom` are only useful when using the `customize` interface.

Reply:
Ah, I didn't realize this was a previously available option. You may disregard my commentary :)

Reply:
Anyway thanks for reviewing that at least :joy: